package-lock.json

```diff
@@ -1844,9 +1844,9 @@
       }
     },
     "tree-kill": {
-      "version": "1.2.1",
-      "resolved": "https://registry.npmjs.org/tree-kill/-/tree-kill-1.2.1.tgz",
-      "integrity": "sha512-4hjqbObwlh2dLyW4tcz0Ymw0ggoaVDMveUB9w8kFSQScdRLo0gxO9J7WFcUBo+W3C1TLdFIEwNOWebgZZ0RH9Q=="
+      "version": "1.2.2",
+      "resolved": "http://localhost:4873/tree-kill/-/tree-kill-1.2.2.tgz",
```

Comment:
Why should it be localhost?

Reply:
Oh sorry. The host is generally ignored - it's an after effect of using [verdaccio](http://npmjs.com/verdaccio) as a local npm cache/proxy.